File: cpt.py
def scalar(vector, factor):
    final_vec = []
    try:
        for i in vector:
            final_vec.append(int(i) * int(factor))
    except:
        return "no floating point values"

    return final_vec

vector = input().split()
factor = input()

print(scalar(vector, factor))

File: cpt.rs
fn scalar(vector: Vec<i32>,factor: i32){
    let mut final_vec :Vec<i32>;
    for number in vector{
        final_vec.append(number * factor);
    }
    return final_vec;
}

fn main() {
    println!("{}", scalar([2,2],5));
}